packages/chrono-core/package.json

```diff
@@ -1,7 +1,11 @@
 {
-  "name": "@neofinancial/chrono-core",
-  "version": "1.0.0",
+  "name": "@neofinancial/chrono",
+  "version": "0.1.0",
   "description": "",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
so normally scoped packages publish privately to the scope. this will allow public publish